src/node/chainstate.h

```diff
@@ -35,7 +36,11 @@ struct ChainstateLoadOptions {
 //! case, and treat other cases as errors. More complex applications may want to
 //! try reindexing in the generic failure case, and pass an interrupt callback
 //! and exit cleanly in the interrupted case.
-enum class ChainstateLoadStatus { SUCCESS, FAILURE, FAILURE_INCOMPATIBLE_DB, INTERRUPTED };
+enum class ChainstateLoadStatus { SUCCESS,
+                                  FAILURE,
+                                  FAILURE_INCOMPATIBLE_DB,
+                                  INTERRUPTED,
+                                  FAILURE_INSUFFICIENT_DBCACHE };
```

Comment:
42402bffda6643346cb2b51c9e85b389f3d19068 nit: Maybe order this before/after `FAILURE_INCOMPATIBLE_DB` to group related things into one section?

Reply:
done